sceptre/bridge/config/develop/BridgeServer2-vpc.yaml

```diff
@@ -1,6 +1,6 @@
 template:
   type: http
-  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.4.0/vpc.yaml
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.4.10/vpc.yaml
```

Comment:
Any concern with the big jump (most of the remaining vpc are on 4.0.0)?

Reply:
looking at the diff, it seems ok to me. 

➜  git diff v0.4.10..v0.4.0 templates/vpc.yaml
```
diff --git a/templates/vpc.yaml b/templates/vpc.yaml
index 7aa40c9..7d82783 100644
--- a/templates/vpc.yaml
+++ b/templates/vpc.yaml
@@ -19,9 +19,9 @@ Parameters:
     ConstraintDescription: List of Availability Zones in a region, such as us-east-1a, us-east-1b, us-east-1c
     Default: "us-east-1a, us-east-1b, us-east-1c"
   VpnCidr:
-    Description: CIDR of the TGW VPN
+    Description: CIDR of the (sophos-utm) VPN
     Type: String
-    Default: "10.50.0.0/16"
+    Default: "10.1.0.0/16"
     AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
     ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
   Department:
@@ -51,18 +51,8 @@ Parameters:
       - true
       - false
     Default: true
-  IncludeBastianSecurityGroup:
-    Type: String
-    Description: >
-      true (default) to deploy a bastian security group
-      false to skip the bastian security group
-    AllowedValues:
-      - true
-      - false
-    Default: true
 Conditions:
   EnableS3GatewayEndpoint: !Equals [!Ref IncludeS3GatewayEndpoint, true]
-  EnableBastianSecurityGroup: !Equals [!Ref IncludeBastianSecurityGroup, true]
 Mappings:
   SubnetConfig:
     VPC:
@@ -479,6 +469,11 @@ Resources:
           ToPort: -1
           IpProtocol: "-1"
           Description: "Allow all VPN traffic"
+        - CidrIp: "10.50.0.0/16"    # AWS TGW Hub
+          FromPort: -1
+          ToPort: -1
+          IpProtocol: "-1"
+          Description: "Allow AWS TGW Hub traffic"
       # CF does not support removing all rules, workaround is to add a pointless rule
       SecurityGroupEgress:
         - CidrIp: "0.0.0.0/0"
@@ -498,7 +493,6 @@ Resources:
   # Allow access to bastian hosts
   BastianSecurityGroup:
     Type: 'AWS::EC2::SecurityGroup'
-    Condition: EnableBastianSecurityGroup
     Properties:
       GroupDescription: Security Group for bastians
       VpcId:
@@ -649,7 +643,6 @@ Outputs:
         !Join ['-', [!Ref 'AWS::Region', !Sub '${AWS::StackName}', 'VpnSecurityGroup']]
   BastianSecurityGroup:
     Description: "Bastian Security Group Id"
-    Condition: EnableBastianSecurityGroup
     Value: !Ref BastianSecurityGroup
     Export:
       Name:
```